src/vs/platform/actions/common/actions.ts

```diff
@@ -435,6 +435,7 @@ export class MenuItemAction implements IAction {
 	readonly id: string;
 	readonly label: string;
 	readonly tooltip: string;
+	readonly description?: ILocalizedString | undefined;
```

Comment:
We have not been exposing `ICommandAction` specific properties on this types. It's mostly constrained to `IAction` and extended propries are accessed via `.item.XYZ`. I'd suggest that `description` also follows this model

Reply:
done